File: Ex-28/ex_28_java/Ponto.java
public class Ponto {
    private float x;
    private float y;

    public Ponto(float x, float y) {
        this.x = x;
        this.y = y;
    }

    public Ponto() {
        this(0.0f, 0.0f);
    }

    public void setPonto(float x, float y) {
        this.x = x;
        this.y = y;
    }

    public float getX() {
        return x;
    }

    public float getY() {
        return y;
    }
}

File: Ex-28/ex_28_python/ponto.py
class Ponto:
    def __init__(self, x=0.0, y=0.0):
        self.x = x
        self.y = y

    def __del__(self):
        pass

    def setPonto(self, x, y):
        self.x = x
        self.y = y

    def getX(self):
        return self.x

    def getY(self):
        return self.y
